### DISCUSSION TITLE
Query rich text field on postgres database

### DISCUSSION THREAD

**Discussion by a-dmx** [payloadcms/payload#5575] [Q&A]
I have a question how to query the rich text  field on postgres database.

`
                const offers = await payload.find({
                    collection: 'offers',
                    depth: 3,
                    where: {
                            {
                                'description': {
                                    like: searchText
                               }
                    }
                })
`

When I execute the above I get the following error:
error: operator does not exist: jsonb ~~* unknown

**Comment by DanRibbens** [comment]
I thought that we handled this but maybe we're missing something in our handling of the search value you're using.

Can you give an example of the value you might have in `searchText`, you're not using any special characters or anything are you?

This is where we handle querying:
[packages/db-postgres/src/queries/parseParams.ts](https://github.com/payloadcms/payload/blob/02d2c517176a775c6eeb4b164d9e6b45d8f5e4c1/packages/db-postgres/src/queries/parseParams.ts#L106-147)

If you see anything that can be improved, feel free to open a PR!
